src/main/scala-2.11/com/linagora/gatling/imap/protocol/command/StoreHandler.scala

```diff
@@ -0,0 +1,64 @@
+package com.linagora.gatling.imap.protocol.command
+
+import java.util
+
+import akka.actor.{ActorRef, Props}
+import com.lafaspot.imapnio.client.IMAPSession
+import com.lafaspot.imapnio.listener.IMAPCommandListener
+import com.linagora.gatling.imap.protocol.{Command, ImapResponses, Response, Tag}
+import com.sun.mail.imap.protocol.IMAPResponse
+import io.gatling.core.akka.BaseActor
+
+import scala.collection.immutable.Seq
+
+abstract class StoreFlags(prefix: String, silent: Boolean, flags: String*) {
+  def asString: String = prefix + (if (silent) ".SILENT" else "") + " " + flags.mkString("(", " ", ")")
+}
+
+object StoreFlags {
+  case class FlagReplace(silent: Boolean, flags: String*) extends StoreFlags("FLAGS", silent, flags:_*)
+
+  case class FlagAdd(silent: Boolean, flags: String*) extends StoreFlags("+FLAGS", silent, flags:_*)
+
+  case class FlagRemove(silent: Boolean, flags: String*) extends StoreFlags("+FLAGS", silent, flags:_*)
+
+}
+
+object StoreHandler {
+  def props(session: IMAPSession, tag: Tag) = Props(new StoreHandler(session, tag))
+}
+
+class StoreHandler(session: IMAPSession, tag: Tag) extends BaseActor {
+
+  override def receive: Receive = {
+    case Command.Store(userId, sequence, flags) =>
+      val listener = new StoreListener(userId)
+      val sequenceAsString = sequence.map(_.asString).mkString(",")
+      val attributesAsString = flags.asString
+      session.executeTaggedRawTextCommand(tag.string, s"STORE $sequenceAsString $attributesAsString", listener)
```

Comment:
why do we use raw commands for everything ? IMAPSession probably has store and expunge, isn't it ?